dipy/denoise/pca_noise_estimate.pyx

```diff
@@ -92,18 +100,32 @@ def pca_noise_estimate(data, gtab, patch_radius=1, correct_bias=True,
     if (dsm != 1) and (dsm < 2 * patch_radius + 1):
         raise ValueError("Array 'data' is incorrect shape")
 
-    X = data0.reshape(nsamples, n3)
+    if images_as_samples:
+        X = data0.reshape(nsamples, n3).T
+    else:
+        X = data0.reshape(nsamples, n3)
+
     # Demean:
     M = np.mean(X, axis=0)
     X = X - M
     U, S, Vt = svd(X, *svd_args)[:3]
     # Rows of Vt are the eigenvectors, in ascending eigenvalue order:
     W = Vt.T
-    # Project into the data space
-    V = X.dot(W)
 
-    # Grab the column corresponding to the smallest eigen-vector/-value:
-    I = V[:, -1].reshape(n0, n1, n2)
+    if images_as_samples:
+        # use second-to-last eigenvector, because of X is centered
+        W = W.astype('double')
+        V = W[:, n3-2].reshape(n0, n1, n2)
+
+        # eigenvalue = variance gives scale (ref [1]_ method is ambigious)
```

Comment:
Not sure if I understand the comment.

Reply:
Basically, the original paper is extremely unclear on a certain point. I emailed them, to no avail. I will try to find a better worded comment, but the current code (i.e. with my changes for estimating the noise level) does seem to estimate noise properly

Reply:
> I emailed them, to no avail

Thanks for the effort Sam.

Reply:
Although I have tested both the canonical (newly added) and non-canonical (originally in DiPy) code and found they perform extremely similarly, I have decided to leave the original algorithm as the default, and have improved these comments, and the documentation, to explain.

Reply:
OK. As said in the related above comment, other reviewers might want to weigh in.